cmd/run.go

```diff
@@ -103,7 +104,7 @@ var runCmd = &cobra.Command{
 		languageDetector := lsp.NewFileExtensionBasedLanguageDetector()
 		diagnosticsStore := lsp.NewDiagnosticsStore()
 		clientPool := lsp.NewClientPool()
-		lspService := lsp.NewService(languageDetector, lspServerExecutables, diagnosticsStore, clientPool)
+		lspService := lsp.NewService(languageDetector, lspServerExecutables, diagnosticsStore, clientPool, []protocol.SymbolKind{protocol.SymbolKindField, protocol.SymbolKindVariable})
```

Comment:
I think those options should be passed to `SearchSymbols()` by the caller func. This should add some flexibility to the codebase.

Reply:
Done